### PR TITLE
Just like back to the days we were using changelog typewriter monkeys.

### PR DESCRIPTION
**Pull request by Ghommie**
If you couldn't have guessed from the jestful title, this PR ascends DM;CA's changelog up to the current day.
If you have any suggestion, complaint, thing to say, do review or comment. I'm pretty damn sure I made numerous typos, improper usage of past simple, word salading and may have missed a merged PR or two.

Html is painfully boring.

### CODE REVIEW

**Comment by F-OS** [comment]
how do i upvote a PR 15 times

File: html/changelog.html [hunk 1]
```diff
@@ -24,27 +24,28 @@
 <table align='center' class="top">
 	<tr>
 		<td valign='top'>
-			<div align='center'><font size='3'><b>Welcome to Colonial Marines!</b></font></div><br>
-			<div align='center'><font size='2'><i>Colonial Marines is a medium-roleplay Space Station 13 server inspired by the Aliens film and built from a heavily modified Baystation 12.</i></font></div>
+			<div align='center'><font size='3'><b>Welcome to DM;CA!</b></font></div><br>
+			<div align='center'><font size='2'><i>Digital Marines ; Colonel Alien is a low-medium roleplay Space Station 13 server branched off CM-SS13, which branched off Baystation 12, which branched off /tg/Station 13.</i></font></div>
 			<br>
-			<div align='center'><font size='3'><a href="https://cm-ss13.com/viewforum.php?f=82">Rules</a> | <a href="https://cm-ss13.com/">Forum</a> | <a href="https://cm-ss13.com/wiki">Wiki</a><br></font></div>
```
Comment (F-OS) [inline html/changelog.html]: I think we're using https://tgstation13.org/wiki/DMCA_Rules instead of the regular rules and the Addendum

Comment (Ghommie) [reply html/changelog.html]: We are mainly following /tg/station ruleset, the addendum url only contain a few indications specific to this server, with no citation or direct link to the other rules. Better keep this safe and square.

Comment (F-OS) [reply html/changelog.html]: understandable! Thanks!

**Pull request merged**